t/04-scheduler.t

```diff
@@ -294,9 +294,7 @@ subtest 'job listing' => sub {
 
 # assume the worker has just been seen
 my $last_seen = DateTime->now(time_zone => 'UTC');
-$last_seen->subtract(seconds => DB_TIMESTAMP_ACCURACY);
-$worker_db_obj->update({t_seen => $last_seen});
-$worker_db_obj->discard_changes;
+$last_seen = $worker_db_obj->update({t_seen => $last_seen})->t_seen;
```

Comment:
Please keep subtracting one second. Otherwise the further checks don't work: We want to assert here that `t_seen` has not been updated. However, now we wouldn't notice anymore unless a second has actually passed.

Reply:
I was indeed curious to see if it makes a difference. Locally not subtracting seems to make it flaky

Reply:
> Locally not subtracting seems to make it flaky

Yes. That's the check which actually asserts that an update has happened. It becomes flaky for the same reason the other checks won't work anymore.